apps/common-app/package.json

```diff
@@ -45,6 +45,7 @@
     "@react-navigation/native": "^6.1.9",
     "@react-navigation/native-stack": "^6.9.17",
     "@react-navigation/stack": "^6.3.18",
+    "@shopify/flash-list": "^1.6.4",
```

Comment:
Not sure if we would like to revert `FlashList` example for now and wait until the new version, supporting fabric, is released.

Reply:
Yes, if they include [your PR](https://github.com/Shopify/flash-list/pull/1041) that adds `getNativeScrollRef` in the next release.